bin/rails

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env ruby
 APP_PATH = File.expand_path('../config/application', __dir__)
-require_relative '../config/boot'
-require 'rails/commands'
+require_relative "../config/boot"
```

Comment:
These are the new defaults in Rails :grimacing: 

It's probably stupid to change them here, but I think it would be easier to track changes in the default files when doing `rails app:update` in the future.

There are a few related PRs... 
https://github.com/rails/rails/pull/42064
https://github.com/rails/rails/pull/41733
https://github.com/rails/rails/pull/41080